python/dllib/src/bigdl/dllib/utils/file_utils.py

```diff
@@ -13,16 +13,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-from bigdl.dllib.utils.common import Sample as BSample, JTensor as BJTensor,\
-    JavaCreator, _get_gateway, _py2java, _java2py
-import numpy as np
+
+import functools
+import glob
 import os
+import subprocess
 import tempfile
+from urllib.parse import urlparse
 import uuid
-import functools
-import glob
 
-from urllib.parse import urlparse
+import numpy as np
+import pyarrow as pa
```

Comment:
put this import locally in the function? as dllib is not necessarily depends on pyarrow

Reply:
fixed